client/galaxy/scripts/components/admin/Jobs.vue

```diff
@@ -129,8 +129,8 @@
                         :value="data.item['job_info']['id']"
                     ></b-form-checkbox>
                 </template>
-                <template v-slot:cell(job_info)="data">
```

Comment:
I didn't seen you also tried to merge 19.09 forward and opened https://github.com/galaxyproject/galaxy/pull/9104 . I resolved the conflict here differently, see https://github.com/galaxyproject/galaxy/pull/9104/files#diff-8fd44636765df0b938c8ff566954e688

Reply:
Ah, yeah, comparing that with dannon's PR, I think your variant is correct.